scripts/build_osm_network.py

```diff
@@ -639,6 +761,10 @@ def fix_overpassing_lines(lines, buses, distance_crs, tol=1):
         below which the line will be splitted
     """
 
+    # in case of @lines corresponding to links it may be an empty data frame
+    if len(lines) == 0:
```

Comment:
maybe lines.empty is more appropriate?

Reply:
I think we can safely remove this check as `lines` here is a merged ac/dc dataframe which shouldn't be empty.

Reply:
I agree, this check could be removed